### DISCUSSION TITLE
enum配置的问题

### DISCUSSION THREAD

**Discussion by henryzp** [alibaba/formily#1741] [Q&A 1.x]
想了解一下，就是当enum的dataSource，label为jsx时，鼠标移上去的时候会显示[Object object]

有没有配置title的地方，尝试用了title，不行

![image](https://user-images.githubusercontent.com/3849702/124740059-fcfed600-df4c-11eb-9c70-c81587bb7bd2.png)

**Comment by henryzp** [comment]
我的label是jsx的写法：

![image](https://user-images.githubusercontent.com/3849702/124740164-1dc72b80-df4d-11eb-8cf8-ab3752142b85.png)

**Comment by janryWang** [comment]
这是antd的问题吧

**Reply by henryzp** [reply]
看起来是的，暂时好像无解
